server/http.go

```diff
@@ -354,22 +354,12 @@ func (p *Plugin) completeUserOAuthToZoom(w http.ResponseWriter, r *http.Request)
 	}
 
 	p.trackConnect(userID)
-
 	if justConnect {
 		p.postEphemeral(userID, channelID, "", "Successfully connected to Zoom")
 	} else {
-		meeting, err := client.CreateMeeting(zoomUser, defaultMeetingTopic)
-		if err != nil {
-			p.API.LogWarn("Error creating the meeting", "error", err.Error())
-			return
-		}
-
-		meetingID := meeting.ID
-		if err = p.postMeeting(user, meetingID, channelID, "", ""); err != nil {
-			p.API.LogWarn("Failed to post the meeting", "error", err.Error())
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-			return
-		}
+		// Ignoring the returned error here as we are already logging it in the function.
```

Comment:
Can we instead log the error here? This sort of comment seems a little out of place